auslib/test/admin/views/test_releases.py

```diff
@@ -1033,6 +1033,8 @@ class TestReleasesScheduledChanges(ViewTest):
 
     def setUp(self):
         super(TestReleasesScheduledChanges, self).setUp()
+        # TODO: Although sc_id has a signoff, it's actually not required. We should have a test
+        # that actually tests a scheduled change that requires a signoff
```

Comment:
Was this meant for test_required_signoffs.py? I just realized there's no follow-up to this in test_releases.py.